marker_api/src/ast/item/use_item.rs

```diff
@@ -19,6 +19,11 @@ use super::CommonItemData;
 /// // `use_kind()` -> `Single`
 /// ```
 ///
+/// There is a separate [`UseItem`] per each path in a `use` declaration.
+/// For example, if you have `use foo::{bar, baz};` there will be two [`UseItem`]s
+/// one for `bar` and one for `baz` and they will share a prefix in their
+/// [`UseItem::use_path()`].
```

Comment:
Nice catch. I thought, I had already documented this behavior, but apparently not. Before 1.0 we might want to revisit this representation, but for now, this is a solid improvement :D